sklearn/metrics/classification.py

```diff
@@ -372,7 +372,8 @@ class labels [2]_.
     return 1 - k
 
 
-def jaccard_similarity_score(y_true, y_pred, normalize=True,
+def jaccard_similarity_score(y_true, y_pred, labels=None, pos_label=1,
+                             average='samples', normalize=None,
```

Comment:
I think normalize should default to true

Reply:
What I intended was, though `normalize=None` is default in API but it behaves by default as `normalize=True`. With `normalize=None` in API, it will give us a possibility of raising error, for when user explicitly sets `normalize` as a bool value.

Like `jaccard_similarity_score(y_true, y_pred, normalize=True, average='weighted')`, we can make it to raise error here with `normalize=None` as default in API, but not when `normalize` is bool in API.

WDYT?